miner/proposal_builder.go

```diff
@@ -368,6 +373,20 @@ func (pb *ProposalBuilder) decideMeshHash(ctx context.Context, current types.Lay
 	return mesh
 }
 
+func (pb *ProposalBuilder) UpdateActiveSet(epoch types.EpochID, activeSet []types.ATXID) {
```

Comment:
what about also providing weight as a part of fallback data?

Reply:
We could, I wanted to avoid that because for this we have to change the schema of the json thats inside the bootstrap file - causing all nodes to fail to validate it unless they update.